windows/src/engine/keyman/UfrmKeymanMenu.pas

```diff
@@ -42,14 +42,33 @@
 interface
```

Comment:
There are a lot of changes in this file. The diff may be confusing. It may be better to read it as a whole new file.